app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
@kbrock I hope I understand you comment

Turns out in ruby:
```ruby
def strange_example(*a,b,c)
  puts "[#{a}] [#{b}] [#{c}]"
end

> strange_example(1,2,3,4)
[[1, 2]] [3] [4]

> strange_example(1,2)
[[]] [1] [2]
```

Also I tested this code (which wasn't introduced here).